src/common/hooks/useEndorsementChain/useEndorsementChain.ts

```diff
@@ -36,11 +36,8 @@ export const useEndorsementChain = (
       const transferEvents = mergeTransfers([...titleEscrowLogs, ...tokenLogs]);
       const retrievedEndorsementChain = await getEndorsementChain(provider, transferEvents);
       setEndorsementChain(retrievedEndorsementChain);
-    } catch (e) {
-      if (e instanceof Error) {
-        console.error(e);
-        setError(e.message);
-      }
+    } catch (e: any) {
+      setError(e?.message);
```

Comment:
is there an error that was missed out of instanceof Error? 
can take a look at this article: https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript

best is to dont use `any` in typescript